causal_velocity_filter.py

```diff
@@ -8,27 +9,56 @@ class CausalVelocityFilter():
     derivative with window size defined by the init.
     """
 
-
-    def __init__(self, windowSize):
+    def __init__(self,
+                 sampling_period,
+                 window_length=99,
+                 polynomial_order=5,
```

Comment:
this is a very high polynomial order.  Why so high?

Reply:
From what I read in *Numerical Recipes*, this is about typical for this filter.  But ultimately, I could probably change the polynomial and it'd be fine.